frontend/test/metabase/scenarios/setup/setup.cy.spec.js

```diff
@@ -1,143 +1,162 @@
 import path from "path";
 import { restore } from "__support__/cypress";
 
+// we're testing for one known (en) and one unknown (xx) locale
+const locales = ["en", "xx"];
+
 describe("scenarios > setup", () => {
-  before(() => restore("blank"));
-
-  it("should allow you to sign up", () => {
-    // intial redirection and welcome page
-    cy.visit("/");
-    cy.url().should("be", "/setup");
-    cy.findByText("Welcome to Metabase");
-    cy.findByText("Let's get started").click();
-
-    // ========
-    // Language
-    // ========
-
-    cy.findByText("What's your preferred language?");
-    cy.findByText("English").click();
-    cy.findByText("Next").click();
-
-    // ====
-    // User
-    // ====
-
-    // "Next" should be disabled on the blank form
-    // NOTE: unclear why cy.findByText("Next", { selector: "button" }) doesn't work
-    // alternative: cy.contains("Next").should("be.disabled");
-    cy.findByText("Next")
-      .closest("button")
-      .should("be.disabled");
-
-    cy.findByLabelText("First name").type("Testy");
-    cy.findByLabelText("Last name").type("McTestface");
-    cy.findByLabelText("Email").type("testy@metabase.com");
-    cy.findByLabelText("Your company or team name").type("Epic Team");
-
-    // test first with a weak password
-    cy.findByLabelText("Create a password").type("password");
-    cy.findByLabelText("Confirm your password").type("password");
-
-    // the form shouldn't be valid yet and we should display an error
-    cy.findByText("must include one number", { exact: false });
-    cy.findByText("Next")
-      .closest("button")
-      .should("be.disabled");
-
-    // now try a strong password that doesn't match
-    const strongPassword = "QJbHYJN3tPW[";
-    cy.findByLabelText("Create a password")
-      .clear()
-      .type(strongPassword);
-    cy.findByLabelText("Confirm your password")
-      .clear()
-      .type(strongPassword + "foobar")
-      .blur();
-
-    // tell the user about the mismatch after clicking "Next"
-    cy.findByText("Next")
-      .closest("button")
-      .should("be.disabled");
-    cy.findByText("passwords do not match", { exact: false });
-
-    // fix that mismatch
-    cy.findByLabelText("Confirm your password")
-      .clear()
-      .type(strongPassword);
-
-    // Submit the first section
-    cy.findByText("Next").click();
-
-    // ========
-    // Database
-    // ========
-
-    // The database step should be open
-    cy.findByText("You’ll need some info about your database", {
-      exact: false,
+  locales.forEach(locale => {
+    beforeEach(() => restore("blank"));
+
+    it(`should allow you to sign up using "${locale}" browser locale`, () => {
+      // TEMP replacement for `it.skip()`
+      // should be removed as soon as fix for Issue #13045 is merged
+      if (locale === "xx") {
+        return;
+      }
```

Comment:
@paulrosenzweig this is probably very cheap/lame "hack", but it seemed like the easiest solution to conditionally skip the test within `forEach()` loop. Despite being fragile, I hope it's ok as a temporary code that will be removed as soon as your fix (#13048) is fully merged?